Production/govuk_ios/Builders/CoordinatorBuilder.swift

```diff
@@ -1,12 +1,29 @@
 import UIKit
 import Foundation
 
+import Resolver
```

Comment:
Why Resolver and not [Factory](https://github.com/hmlongco/Factory) Tom? 

I understand that Resolver is officially deprecated in favour of Factory and they are both written by Michael Long.

Reply:
Absolutely no reason, just blindly added the one that a couple of us have used in the past. Good spot. I will swap out resolver for factory... good test to see if I have written it well enough to swap libs 👍🏻